test/src/waittask.spec.ts

```diff
@@ -212,46 +174,6 @@ describe('waittask specs', function () {
       ]);
       expect(error).toBeUndefined();
     });
-    it('should throw on bad polling value', async () => {
-      const {page} = getTestState();
-
-      let error!: Error;
-      try {
-        await page.waitForFunction(
-          () => {
-            return !!document.body;
-          },
-          {
-            polling: 'unknown',
-          }
-        );
-      } catch (error_) {
-        if (isErrorLike(error_)) {
-          error = error_ as Error;
-        }
-      }
-      expect(error?.message).toContain('polling');
-    });
-    it('should throw negative polling interval', async () => {
```

Comment:
let's keep this test?

Reply:
We don't provide checks for these anymore, we just leave it UB.

Reply:
why not keep the validation? seems to be a better UX if the users by some chance provide negative values.